installing/installing_sno/install-sno-installing-sno.adoc

```diff
@@ -72,6 +88,33 @@ include::modules/installation-aws_con_installing-sno-on-aws.adoc[leveloffset=+2]
 
 include::modules/install-sno-installing-with-usb-media.adoc[leveloffset=+1]
 
-include::modules/install-booting-from-an-iso-over-http-redfish.adoc[leveloffset=+1]
+[id="installing-sno-with-ibmpower"]
```

Comment:
@kdrake20 it looks like you pulled in some additional content.

Reply:
Updated As Suggested.